--- ./usr/local/lib/python3.12/dist-packages/poxy/run.orig	2024-11-10 19:50:41.858752777 +0000
+++ ./usr/local/lib/python3.12/dist-packages/poxy/run.py	2024-11-10 19:50:50.478776979 +0000
@@ -746,7 +746,7 @@
 
                     # re-sort members to override Doxygen's weird and stupid sorting 'rules'
                     if 1:
-                        sort_members_by_name = lambda tag: tag.find(r'name').text
+                        sort_members_by_name = lambda tag: tag.find(r'name').text or ''
                         members = [tag for tag in section.findall(r'memberdef')]
                         for tag in members:
                             section.remove(tag)
